### DISCUSSION TITLE
Execute extension on startup

### DISCUSSION THREAD

**Discussion by rwxroot** [Ulauncher/Ulauncher#1361] [Q&A] [answered]
Is there a way to execute an extension on startup? I would like the ability to run it when the window pops up, similar to the frequent apps list.

**Comment by friday** [comment, marked as answer]
In the v6 beta you can bind a different hotkey to: `gapplication action io.ulauncher.Ulauncher set-query "'kw '"` (where kw is your extension keyword)

**Reply by rwxroot** [reply]
That sounds delightful. Thanks a ton, appreciate your quick response!
